docs/release-notes/4257-which.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Fixes**
+
+-  Support running in containers which did not have ``/bin/which``, like python-slim. The error was
```

Comment:
maybe just "Don't require task image's `which` to be located in `/bin/which`"

Reply:
@gh-determined-ai do you want this detail or not?

(@stoksc I frequently get asked to add detail to the release notes I write)